### PR TITLE
Book Detail III - Structure

### PR DESCRIPTION
**Pull request by seromarin**
## Summary
- Completed detail book structure
- **CSS unit sizes fixed**

## Screenshot
Captura de Pantalla 2020-10-30 a la(s) 2 39 17 p  m

## Trello Card
https://trello.com/c/WvLbEmRU/28-detalle-de-libro-iii-estructura

### CODE REVIEW

File: maquetacion/src/scss/_detail.scss [hunk 1]
```diff
@@ -0,0 +1,50 @@
+.detail {
+  background-color: #FFF;
```
Comment (mcallegari10) [inline maquetacion/src/scss/_detail.scss:2]: Use variables for colors please 😄

File: maquetacion/src/scss/_detail.scss [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  box-shadow: 4px 4px 10px 0 rgba(0,0,0,0.5);
```
Comment (mcallegari10) [inline maquetacion/src/scss/_detail.scss:3]: Same here, variable for the color

File: maquetacion/src/scss/_detail.scss [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+  &__cover-img {
+    height: 368px;
+    width: 261px;
+  }
+
+  &__genre {
+    height: 30px;
+    width: 98px;
+    font-family: Roboto;
```
Comment (mcallegari10) [inline maquetacion/src/scss/_detail.scss:25]: I think everything has Roboto, if not remember to set a backup like `sans-serif`

File: maquetacion/src/scss/_detail.scss [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    font-size: 24px;
+    font-weight: 700;
+    line-height: 28px;
+  }
+
+  &__header {
+    padding: 15px 0;
+    border-bottom: 4px solid #BED23C;
```
Comment (mcallegari10) [inline maquetacion/src/scss/_detail.scss:33]: Add variable please

File: maquetacion/src/scss/_detail.scss [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+    width: 210px;
+    font-size: 32px;
+    font-weight: 700;
+    line-height: 38px;
+  }
+
+  &__author,
+  &__editorial,
+  &__year {
+    margin: 2rem 0;
```
Comment (mcallegari10) [inline maquetacion/src/scss/_detail.scss:47]: px 😄

**Review by mcallegari10 — CHANGES_REQUESTED** [review]

File: maquetacion/src/scss/_detail.scss (hunk 1 above)
Comment (seromarin) [reply maquetacion/src/scss/_detail.scss:2]: Fixed in last branch / commit

File: maquetacion/src/scss/_detail.scss (hunk 2 above)
Comment (seromarin) [reply maquetacion/src/scss/_detail.scss:3]: Fixed in last branch / commit

File: maquetacion/src/scss/_detail.scss (hunk 3 above)
Comment (seromarin) [reply maquetacion/src/scss/_detail.scss:25]: Fixed in last branch / commit

File: maquetacion/src/scss/_detail.scss (hunk 4 above)
Comment (seromarin) [reply maquetacion/src/scss/_detail.scss:33]: Fixed in last branch / commit

File: maquetacion/src/scss/_detail.scss (hunk 5 above)
Comment (seromarin) [reply maquetacion/src/scss/_detail.scss:47]: Fixed in last branch / commit

File: maquetacion/src/scss/_detail.scss [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+    width: 100%;
+  }
+
+  &__cover {
+    margin-right: 52px;
+  }
+
+  &__cover-img {
+    height: 368px;
+    width: 261px;
```
Comment (idcuadrowolox) [inline maquetacion/src/scss/_detail.scss:19]: Width or height..

**Review by idcuadrowolox — APPROVED** [review]

File: maquetacion/src/scss/_detail.scss (hunk 6 above)
Comment (seromarin) [reply maquetacion/src/scss/_detail.scss:19]: Image only with width

**Review by seromarin — COMMENTED** [review]
All comments fixed here or later in another commit/branch

**Pull request merged**